sos/plugins/kubernetes.py

```diff
@@ -22,33 +22,75 @@ class kubernetes(Plugin, RedHatPlugin):
     """Kubernetes plugin
     """
 
-    option_list = [("podslog", "capture logs for pods", 'slow', False)]
+    packages = ('kubernetes',)
```

Comment:
Really? Is that the actual package that provides the kubernetes commands in Atomic?